src/nidigital/examples/nidigital_ppmu_source_and_measure.py

```diff
@@ -0,0 +1,100 @@
+#!/usr/bin/python
+
+import argparse
+import nidigital
+import os
+import sys
+import time
+
+
+def example(resource_name, options, channels, aperture_time, settling_time, current_level_range, current_level,
+            voltage_limit_high, voltage_limit_low):
+
+    with nidigital.Session(resource_name=resource_name, options=options) as session:
+
+        dir = os.path.join(os.path.dirname(__file__), 'ppmu_source_and_measure_files')
+
+        # Load pin map (.pinmap) created using Digital Pattern Editor
+        pin_map_filename = os.path.join(dir, 'PinMap.pinmap')
+        session.load_pin_map(pin_map_filename)
+
+        # Configure PPMU measurements
+        session.channels[channels].ppmu_aperture_time = aperture_time
+        session.channels[channels].ppmu_aperture_time_units = nidigital.PPMUApertureTimeUnits.SECONDS
+
+        session.channels[channels].ppmu_output_function = nidigital.PPMUOutputFunction.CURRENT
+
+        session.channels[channels].ppmu_current_level_range = current_level_range
+        session.channels[channels].ppmu_current_level = current_level
+        session.channels[channels].ppmu_voltage_limit_high = voltage_limit_high
+        session.channels[channels].ppmu_voltage_limit_low = voltage_limit_low
+
+        session.channels[channels].ppmu_source()
+
+        # Settling time between sourcing and measuring
+        time.sleep(settling_time)
+
+        current_measurements = session.channels[channels].ppmu_measure(nidigital.PPMUMeasurementType.CURRENT)
+        voltage_measurements = session.channels[channels].ppmu_measure(nidigital.PPMUMeasurementType.VOLTAGE)
+
+        pin_info = session.channels[channels].get_pin_results_pin_information()
+
+        print('{:,<6} {:,<20} {:,<10} {:,<10}'.format('Site', 'Pin Name', 'Current', 'Voltage'))
```

Comment:
Nitpick: Why use bunch of commas between the header columns?

Reply:
Because that is what DCPower's example did. I can remove it.